terminator/lib/queue-handlers.js

```diff
@@ -150,24 +139,33 @@ export async function handleTransactionRetryQueueMessage(
       `Calculating gas fees from the recent Send message ${recentSendMessage.cid}`,
     )
 
+    const originalMaxPriorityFeePerGas = originalTx.maxPriorityFeePerGas
+    assert(
+      originalMaxPriorityFeePerGas !== undefined,
+      'originalTx.maxPriorityFeePerGas is null',
+    )
+
     // Increase by 25% + 1 attoFIL (easier: 25.2%) and round up
     const newMaxPriorityFeePerGas =
-      (originalTx.maxPriorityFeePerGas * 1252n + 1000n) / 1000n
+      (originalMaxPriorityFeePerGas * 1252n + 1000n) / 1000n
 
     const newGasLimit = BigInt(
       Math.min(
         Math.ceil(
-          Math.max(Number(originalTx.gasLimit), recentSendMessage.gasLimit) *
-            1.1,
+          // THIS REQUIRES CAREFUL REVIEW
+          // Viem does not provide `originalTx.gasLimit`, only `gas`.
+          // Is this new calculation still correct?
+          Math.max(Number(originalTx.gas), recentSendMessage.gasLimit) * 1.1,
```

Comment:
Viem does not provide `originalTx.gasLimit`, only `gas`. Is this new calculation still correct?

@pyropy Do you know what the correct solution is here?